charts/plane-enterprise/templates/config-secrets/postgres-init.yaml

```
@@ -0,0 +1,16 @@
+  namespace: {{ .Release.Namespace }}
+  name: {{ .Release.Name }}-pgdb-init
+data:
+  init-databases.sql: |
+    -- Create plane_pi database for PI service
+    CREATE DATABASE plane_pi;
+    
+    -- Grant privileges to the postgres user
+    GRANT ALL PRIVILEGES ON DATABASE plane_pi TO {{ .Values.env.pgdb_username | default "plane" }};
+---
+{{- end }}
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)